homeassistant/components/deluge/config_flow.py

```diff
@@ -86,7 +89,8 @@ async def validate_input(self, user_input: dict[str, Any]) -> str | None:
             await self.hass.async_add_executor_job(api.connect)
         except (ConnectionRefusedError, TimeoutError, SSLError):
             return "cannot_connect"
-        except Exception as ex:  # noqa: BLE001
+        except Exception as ex:
+            _LOGGER.exception("Unexpected error")
             if type(ex).__name__ == "BadLoginError":
                 return "invalid_auth"
             return "unknown"
```

Comment:
I am genuinely wondering why we don't do `except BadLoginError`. I also could not find a reference in the library. @tkdrob Can you maybe give a hint on this?

Reply:
Seems maybe it was still supporting py2 at the time this was done.
Could be fixed now I guess.